scripts/romanisim-make-image

```diff
@@ -62,11 +64,11 @@ if __name__ == '__main__':
     if args.radec is not None:
         coord = SkyCoord(ra=args.radec[0] * u.deg, dec=args.radec[1] * u.deg,
                          frame='icrs')
-        wcs.fill_in_parameters(metadata, coord, boresight=args.boresight)
+        wcs.fill_in_parameters(metadata, coord, boresight=args.boresight, roll_ref=args.roll)
 
     if args.date is not None:
-        metadata['exposure']['start_time'] = Time(
-            datetime.datetime(*args.date)).isot
+        metadata['exposure']['start_time'] = Time(Time(
+            datetime.datetime(*args.date)).isot, format="isot", scale="utc")
```

Comment:
What was the issue with the previous implementation here?  If the issue is that a we'd rather have a Time object than a string in the metadata, then I think you could just drop the '.isot' rather than doubly wrap?